### PR TITLE
[PHEE-395] Added environment variables

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- added env variables for zeebe contactpoint
- added env variables for mock failure percentage
https://fynarfin.atlassian.net/browse/PHEE-395?atlOrigin=eyJpIjoiZTUxZmY2OGRjZWFmNGMzMWJkNjE0MTk1NDQ1YzY0YjciLCJwIjoiamlyYS1zbGFjay1pbnQifQ

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj This pipeline is running from the downstream

File: helm/ph-ee-engine/connector-mock-paymentschema/templates/deployment.yaml [hunk 1]
```diff
@@ -52,8 +52,10 @@ spec:
               value: "{{ .Values.mockpayment.DFSPIDS }}"
             - name: "LOGGING_LEVEL_ROOT"
               value: "{{ .Values.mockpayment.LOGGING_LEVEL_ROOT }}"
+            - name: "ZEEBE_BROKER_CONTACTPOINT"
+              value: "{{ .Release.Name }}-zeebe-gateway:26500"
             - name : "mockFailure_percentage"
-              value: "{{ .Values.mockpayment.mockFailure_percentage }}"
+              value: "{{ .Values.mockFailure_percentage }}"
```
Comment (fynmanoj) [inline helm/ph-ee-engine/connector-mock-paymentschema/templates/deployment.yaml:58]: other fields also requires this fix

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Review by somanath21 — COMMENTED** [review]
I think the previous configuration for "value: "{{ .Values.mockpayment.mockFailure_percentage }}"  is correct, as this service do not have its own values.yaml to refer the value from "value: "{{ .Values.mockFailure_percentage }}"

I believe this is because of missing values in ph-ee-engine/g2p-sandbox values.yaml for mockFailure_percentage

**Review by somanath21 — APPROVED** [review]

File: helm/ph-ee-engine/connector-mock-paymentschema/templates/deployment.yaml [hunk 2]
```diff
@@ -52,8 +52,10 @@ spec:
               value: "{{ .Values.mockpayment.DFSPIDS }}"
             - name: "LOGGING_LEVEL_ROOT"
               value: "{{ .Values.mockpayment.LOGGING_LEVEL_ROOT }}"
+            - name: "ZEEBE_BROKER_CONTACTPOINT"
+              value: "{{ .Release.Name }}-zeebe-gateway:26500"
             - name : "mockFailure_percentage"
-              value: "{{ .Values.mockpayment.mockFailure_percentage }}"
+              value: "{{ .Values.mockpayment.mockFailure.percentage }}"
```
Comment (fynmanoj) [inline helm/ph-ee-engine/connector-mock-paymentschema/templates/deployment.yaml:58]: remove mockpayment from all values
eg: 

value: "{{ .Values.mockFailure.percentage }}"

**Review by fynmanoj — APPROVED** [review]

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**